packages/common/src/core/schemas/hubproject.ts

```diff
@@ -30,6 +30,9 @@ export const HubProjectSchema: IConfigurationSchema = {
     timeline: {
       type: "object",
     },
+    featuredImage: {
```

Comment:
Is this still a top-level thing in the schema? tbh I'm not sure how nesting is handled in here...

Reply:
I was handling things nested in view with logic in the controller upstream, but you raise a good point. I'll discuss w/ @juliannemarik